Dockerfile.openpilot

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:16.04
 ENV PYTHONUNBUFFERED 1
 
 RUN apt-get update && apt-get install -y build-essential clang vim screen wget bzip2 git libglib2.0-0 python-pip capnproto libcapnp-dev libzmq5-dev libffi-dev libusb-1.0-0
-RUN pip install numpy==1.11.2 scipy==0.18.1 matplotlib
+RUN pip install numpy==1.11.2 scipy==0.18.1 matplotlib==2.2.3
```

Comment:
2.1.2 is what's used by our docker

Reply:
What me to change to 2.1.2? I pulled 2.2.3 from latest successful build.

`Collecting matplotlib
  Downloading https://files.pythonhosted.org/packages/59/08/04933377dc4500e3698e93f9113dc3624874e0914f4c85767ecb5b389084/matplotlib-2.2.3-cp27-cp27mu-manylinux1_x86_64.whl (12.6MB)`